### DISCUSSION TITLE
Tutorials: subsection or separate website?

### DISCUSSION THREAD

**Discussion by pervoj** [nahuelwexd/vala-website#38] [Polls]
What do you think is better?

I personally think, that the second option is better, because it could help have things a little bit better organized and also I could start to work on the tutorials in parallel with the creation of the homepage.

It could be then linked as subdomain, e.g. `tutorials.vala.dev`, or `guide.vala.dev`.

**Comment by pervoj** [comment]
So with these results, what do you think is now the best to do? I would probably start to write tutorials in md files and wait for more votes.

**Reply by Suzie97** [reply]
I would vote for a separate website because it would let us completely focus on a tutorial based experience (example: something like GitBook) and that would also make contributions to the documentation easier

**Reply by lw64** [reply]
I am working already since a while on a new documentation website for vala, which should also include tutorials and a contribution guide. So to replace the wiki.
https://lwildberg.pages.gitlab.gnome.org/vala-tutorial/
Here is the repo:
https://gitlab.gnome.org/lwildberg/vala-tutorial

**Reply by pervoj** [reply]
My thoughts were about using MkDocs, since it because to be the best SSG for this type of project.

**Reply by pervoj** [reply]
> I am working already since a while on a new documentation website for vala, which should also include tutorials and a contribution guide. So to replace the wiki. https://lwildberg.pages.gitlab.gnome.org/vala-tutorial/ Here is the repo: https://gitlab.gnome.org/lwildberg/vala-tutorial

Oh nice, it looks like an official GNOME documentation

**Reply by lw64** [reply]
The style is copied from developer.gnome.org. only the color and content is different.

**Reply by lw64** [reply]
For the tutorials I would also link to external resources, like the gnome getting started tutorial, which is translated to vala.

**Reply by Suzie97** [reply]
> The style is copied from developer.gnome.org. only the color and content is different.

Do you think we should use Inter to maintain consistency with the website?

**Reply by lw64** [reply]
I think it looks fine, how it is right now.

**Reply by colinkiama** [reply]
> > The style is copied from developer.gnome.org. only the color and content is different.
> 
> Do you think we should use Inter to maintain consistency with the website?

I think it’s great that they look different. They still share similar colours though.

**Reply by colinkiama** [reply]
So maybe a new site can be made with MkDocs instead? Any other alternatives ideas would be cool too!

**Reply by colinkiama** [reply]
> > The style is copied from developer.gnome.org. only the color and content is different.
> 
> Do you think we should use Inter to maintain consistency with the website?

If we’re using a different solution then I think we should use Inter.

**Reply by colinkiama** [reply]
Here's the site: https://docs.vala.dev

**Comment by Charadon** [comment]
wiki.vala.dev, I think would be a better subdomain, personally. 

I also think it's important that it'd be able to be used without javascript, so screen readers and other accessibility software can parse it more easily.

**Reply by colinkiama** [reply]
Well we've got 2/3 on this list. https://docs.vala.dev
